common/src/misc/iframe_component.ts

```diff
@@ -60,6 +37,22 @@ export class WebAuthn {
         this.win.removeEventListener('message', this.parseFunction, false);
     }
 
+    protected createParams(data: any) {
+        return new URLSearchParams({
+            data: this.base64Encode(JSON.stringify(data)),
+            parent: encodeURIComponent(this.win.document.location.href),
+            v: '1',
```

Comment:
Feels like version should be a property set by the extending class since we probably won't bump version for both at the same time. (My understanding is that it can be used should we ever change the protocol while retaining backward compatibility with existing clients).

Reply:
I guess you're right, but the jslib version and web should be in lock step.